src/shared/route-urls.ts

```diff
@@ -105,6 +105,7 @@ export enum RouteUrls {
   RpcSendTransferSummary = '/send-transfer/summary',
   RpcReceiveBitcoinContractOffer = '/bitcoin-contract-offer/:bitcoinContractOffer/:counterpartyWalletURL',
   RpcSignBip322Message = '/sign-bip322-message',
+  RpcStacksSignature = '/sign-message',
```

Comment:
```suggestion
  RpcStacksSignature = '/sign-stacks-message',
```

to make clear in analytics which type we're signing